recipes/crass/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+./autogen.sh
+./configure --prefix=${PREFIX} --with-xerces=${PREFIX} CPPFLAGS="$CPPFLAGS" LDFLAGS="$LDFLAGS"
```

Comment:
Do you need to specify  `CPPFLAGS="$CPPFLAGS"` on the `configure` command line? It should pick those up from the environment directly...

Reply:
It doesn't seem to pick up environment variables (truth be told most packages don't seem to), as evidenced by some earlier errors.

Reply:
That's curious. It shouldn't matter to `autotools` built things whether the variable comes from the env or the cmdline.